src/mapviewAPI.ts

```diff
@@ -0,0 +1,13 @@
+import { App } from 'obsidian';
+import { GeoSearcher, GeoSearcherProvider } from './geosearch';
+import { PluginSettings } from 'src/settings';
+
+export class MapviewAPI {
```

Comment:
Let's be consistent about the naming, this should be called `MapViewAPI` and the file should be called `mapViewAPI.ts` (capital V)

Reply:
👍